src/cmd-generate-release-meta

```diff
@@ -218,7 +218,8 @@ def append_build(out, input_):
     if i is not None:
         ext = get_extension(i['path'], 'qemu-secex', arch)
         arch_dict["media"]["qemu"]["artifacts"][f"secex.{ext}"] = {
```

Comment:
I suppose this needs to be updated now to say `qemu-secex` given https://github.com/coreos/coreos-assembler/pull/3333.